newrelic/newrelic.ini

```diff
@@ -41,18 +41,15 @@ app_name = Python Application
 # each environment below.
 monitor_mode = true
 
-# Sets the name of a file to log agent messages to. Useful for
-# debugging any issues with the agent. This is not set by
-# default as it is not known in advance what user your web
-# application processes will run as and where they have
-# permission to write to. Whatever you set this to you must
-# ensure that the permissions for the containing directory and
-# the file itself are correct, and that the user that your web
-# application runs as can write to the file. If not able to
-# write out a log file, it is also possible to say "stderr" and
-# output to standard error output. This would normally result in
-# output appearing in your web server log.
-#log_file = /tmp/newrelic-python-agent.log
+# Sets the name of a file to log agent messages to. Whatever you
+# set this to, you must ensure that the permissions for the
+# containing directory and the file itself are correct, and
+# that the user that your web application runs as can write out
+# to the file. If not able to out a log file, it is also
+# possible to say "stderr" and output to standard error output.
+# This is set by default would normally result in output appearing
+# in your web server log.
+log_file = stderr
```

Comment:
Should we not use stdout as the default? If you're going to set one that's what I would set.

Reply:
I'm good with that